### ISSUE TITLE
Batch image detection test with two GPUs

### ISSUE THREAD

**Issue by zhangxue123** [TuSimple/simpledet#212]
How to detect batch image  using two GPUs, please help me, thank you very much!!!

**Comment by RogerChern** [comment]
Do you want to test the model with 2 GPUs?

**Comment by zhangxue123** [comment]
yes, I want to test the model with 2 GPUs, and one GPU can test a batch image, finally, 2 GPUs test result integrate a file or list, can you know how to do it?  @RogerChern

> Do you want to test the model with 2 GPUs?

**Comment by zhangxue123** [comment]
Or test the model with 2 GPUs using a big batch size of image data,  can you know how to do it? @RogerChern

**Comment by zhangxue123** [comment]
Or using a big batch size of image data test the model with 2 GPUs, can you know how to do it? @RogerChern

**Comment by RogerChern** [comment]
The batched test can not be achieved since during the test we do not pad the image. So each image has its unique shape and the 1 image per GPU test scheme is widely adopted by all the detection frameworks.

**Comment by RogerChern** [comment]
Could you please give some details about why batched test is important to you.

**Comment by zhangxue123** [comment]
> Could you please give some details about why batched test is important to you.

I want to statistical and analysis a  result of classification through detection result of  8000+ images, but testing with one image is too slow at a time, and Low resource utilization.
